model/project.go

```diff
@@ -1449,6 +1456,14 @@ func (p *Project) ResolvePatchVTs(bvs, tasks []string, requester, alias string,
 		catcher := grip.NewBasicCatcher()
 		vars, err := FindAliasInProjectOrRepo(p.Identifier, alias)
```

Comment:
I think this function isn't the right thing to do anymore. It doesn't actually just check project/repo, it also checks Project Config (which should probably be part of the title).

Reply:
Modified name to FindAliasInProjectRepoOrConfig

Reply:
Since the current pattern is that aliases in the alias collection take precedence over aliases set in the project configs collection, I think we'd need to check both since they can both potentially contain an active alias.  If someone specifies an alias in their local config for a patch, we can have that take precedence over any alias that is in DB collections since a user would probably want whatever they have locally to be picked up immediately. I'll move that check to be first, and then we'd only check the DB with FindAliasInProjectRepoOrConfig if no matching alias is found in the local config.

Reply:
That behavior seems inconsistent; if in the project ref we have a patch alias called "myAlias' with only one task/variant, and someone tries to locally create an alias called "myAlias" with all tasks for a variant, for their patch they would see all tasks like they expect, but as soon as it's merged the alias will go back to behaving as one task/variant.

Reply:
Gotcha, that makes sense, thanks!